index.html

```diff
@@ -33,11 +33,15 @@
                 brush
                 if-no-vr-headset="visible: false"
                 paint-controls="hand: left"
+                screenshot-camera
+                trigger-mode="brush"
                 ui></a-entity>
       <a-entity id="right-hand"
                 brush
                 if-no-vr-headset="visible: false"
                 paint-controls="hand: right"
+                screenshot-camera
+                trigger-mode="brush"
```

Comment:
Perhaps this should be called action-mode since it might support more than just the trigger button, IE using the grips for scaling instead of undo